GVFS/GVFS.Common/GVFS.Common.csproj

```diff
@@ -28,4 +28,10 @@
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="X509Certificates\" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Security.Cryptography.X509Certificates" />
+  </ItemGroup>

```

Comment:
I don't think any of these lines are needed.  I found that with them the build on Windows complains:

```
2019-01-25T18:13:45.7237754Z   C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "System.Security.Cryptography.X509Certificates". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [C:\agent\_work\11\s\GVFS\GVFS.Common\GVFS.Common.csproj]
2019-01-25T18:13:45.7238315Z   C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3243: No way to resolve conflict between "System.Security.Cryptography.X509Certificates, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Security.Cryptography.X509Certificates". Choosing "System.Security.Cryptography.X509Certificates, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" arbitrarily. [C:\agent\_work\11\s\GVFS\GVFS.Common\GVFS.Common.csproj]
```

And with them removed the build is able to locate the System.Security.Cryptography.X509Certificates assembly.